bibliography.bib

```diff
@@ -1679,3 +1679,33 @@ @manual{HMP2data
   note = {R package version 1.22.0},
   doi = {10.18129/B9.bioc.HMP2Data}
 }
+
+@article{Sprockett2020,
+  author = {Sprockett, D. D. and Martin, M. and Costello, E. K. and others},
+  title = {Microbiota assembly, structure, and dynamics among Tsimane horticulturalists of the Bolivian Amazon},
```

Comment:
Use curly brackets "{}" around letters that should stay capitalized in all renderings (this and check also other references)